### PR TITLE
Added descriptions to arguments listsome

### PR DESCRIPTION
**Pull request by Miguel-F**
Added descriptions to arguments from the Lucee documentation located here - https://docs.lucee.org/reference/functions/listsome.html

### CODE REVIEW

File: data/en/listsome.json [hunk 1]
```diff
@@ -5,15 +5,15 @@
...
-		{"name": "closure", "description": "", "required": true, "default": "", "type": "function", "values": []},
-		{"name": "delimiter", "description": "", "required": false, "default": ",", "type": "string", "values": []},
-		{"name": "includeEmptyFields", "description": "", "required": false, "default": "false", "type": "boolean", "values": []},
-		{"name": "multiCharacterDelimiter", "description": "", "required": false, "default": "true", "type": "boolean", "values": []},
-		{"name": "parallel", "description": "", "required": false, "default": "false", "type": "boolean", "values": []},
-		{"name": "maxThreads", "description": "", "required": false, "default": "20", "type": "numeric", "values": []}
+		{"name": "list", "description": "string list to iterate\n\nAlias: string", "required": true, "default": "", "type": "string", "values": []},
+		{"name": "closure", "description": "function/closure that implements the following constructor [function(any value, numeric index,string list, string delimiter):boolean].\n\nAlias: function, callback, udf", "required": true, "default": "", "type": "function", "values": []},
```
Comment (shaedrich) [inline data/en/listsome.json]: You can use `callback_params` parameter in param to document the closure's params

Comment (Miguel-F) [reply data/en/listsome.json]: I don't know how to do that. Do you have an example?

Comment (shaedrich) [reply data/en/listsome.json]: Sure (more or less the same as normal params):

```
"callback_params": [
    { "values": [], "default": "", "description": "The value of the first key/value pair", "name": "value1", "type": "any", "required": false },
    { "values": [], "default": "", "description": "The value of the second key/value pair", "name": "value2", "type": "any", "required": false }
]
```

**Comment by Miguel-F** [comment]
I'm sure the build failed from my latest edits but I can't tell what the issue is?

**Pull request merged**